isamples_metadata/GEOMETransformer.py

```diff
@@ -404,16 +426,9 @@ def curation_access_constraints(self) -> typing.AnyStr:
         return Transformer.NOT_PROVIDED
 
     def curation_location(self) -> typing.AnyStr:
-        curation_pieces = []
-        tissue_well = self.source_record.get("tissue_well")
-        if tissue_well is not None:
-            curation_pieces.append(f"tissueWell: {tissue_well}")
-        tissue_plate = self.source_record.get("tissue_plate")
-        if tissue_plate is not None:
-            curation_pieces.append(f"tissuePlate: {tissue_plate}")
-        if len(curation_pieces) > 0:
-            return ", ".join(curation_pieces)
-        return Transformer.NOT_PROVIDED
+        return self._source_record_main_record().get(
+            "institutionCode", Transformer.NOT_PROVIDED
+        )
```

Comment:
When I did this initially, I mistakenly put the child code in the main class.  Per https://github.com/isamplesorg/metadata/blob/main/examples/GEOME/GEOMEiSamplesMapping.json, institutionCode is the right place to retrieve this information.

Reply:
ok